src/memcached.c

```diff
@@ -285,6 +297,47 @@ static void submit_gauge2(const char *type, const char *type_inst,
   plugin_dispatch_values(&vl);
 }
 
+static gauge_t calculate_rate(gauge_t part, gauge_t total, gauge_t *prev_part,
```

Comment:
Please consider using `value_to_rate()` to convert both, "part" and "total" to a rate and then simply calculating the ratio as `part_rate / total_rate`.

Reply:
I prefer to leave this untouched. I think use of `value_to_rate()` is excessive for this task.
That adds time calculations, which will be reducted later as the numerator and denominator will have a same value.

Reply:
Fair enough, SGTM